src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -264,10 +270,101 @@ private int LineBreaksAfter(SyntaxToken currentToken, SyntaxToken nextToken)
             return 0;
         }
 
+        private static bool All<TNode>(SyntaxList<TNode> items, Func<TNode, bool> predicate)
+            where TNode : SyntaxNode
+        {
+            bool result = true;
+            for (int i = 0; i < items.Count; i++)
+            {
+                result &= predicate(items[i]);
+            }
+
+            return result;
+        }
+
+        private static bool IsAutoAccessorList(SyntaxToken token)
+        {
+            if (token.Parent is AccessorListSyntax accessorList)
+            {
+                return All(accessorList.Accessors, a => a.Body == null);
+            }
+
+            return false;
+        }
+
+        private static bool IsAutoAccessorList(SyntaxNode? node)
+        {
+            if (node != null && node.Parent != null && node.Parent is AccessorListSyntax accessorList)
+            {
+                return All(accessorList.Accessors, a => a.Body == null);
+            }
+
+            return false;
+        }
+
+        private static bool IsAccessorListFollowedByInitializer(SyntaxToken token)
+        {
+            if (token.Parent != null && token.Parent is AccessorListSyntax accessorList)
+            {
+                var declaration = accessorList.Parent;
+                if (declaration == null)
+                {
+                    return false;
+                }
+
```

Comment:
```suggestion
                var declaration = accessorList.Parent;
```

Reply:
Are you suggestingto remove the null check?
Isn't that dangerous in case the code is in scripting environment without any parent?

Reply:
got it from the other comment below

Reply:
I totally changed it with the `PropertyDeclarationSyntax`